platforms/jvm/jacoco/src/main/java/org/gradle/internal/jacoco/AbstractAntJacocoReport.java

```diff
@@ -51,7 +51,7 @@ public Object doCall(Object it) {
 
     protected void invokeJacocoReport(final GroovyObjectSupport antBuilder, final String projectName,
                      final FileCollection allClassesDirs, final FileCollection allSourcesDirs,
-                     final FileCollection executionData, final T t) {
+                     final String encoding, final FileCollection executionData, final T t) {
```

Comment:
If `encoding` can be null, please add `@Nullable`. Applies throughout

Reply:
fixed